agenta-backend/agenta_backend/models/api/evaluation_model.py

```diff
@@ -234,6 +234,7 @@ class NewEvaluation(BaseModel):
     testset_id: str
     rate_limit: LLMRunRateLimit
     lm_providers_keys: Optional[Dict[LMProvidersEnum, str]]
+    correct_answer_column: Optional[str]
```

Comment:
I would do it like this
```
TestsetSettings(BaseModel):
    correct_answer_column: Optional[str]
    maybe range () in the future or other settings

class NewEvaluation(BaseModel):
    app_id: str
    variant_ids: List[str]
    evaluators_configs: List[str]
    testset_id: str
    testset_settings: TestsetSettings
 ```

Reply:
I would actually only introduce complexity when we need it. That is, if we think we need to add another configuration, we change the interface to add complexity, otherwise, we go with simple first.